Slim/Player/Squeezebox2.pm

```diff
@@ -101,6 +101,10 @@ sub hasScrolling {
 	return shift->revision >= 135;
 }
 
+sub hasBalance {
+	return shift->model(1) =~ /receiver|squeezebox3|transporter/;
```

Comment:
`squeezebox3` is only `squeezebox2` in disguise. No need for the `$wantRealName` flag to `model` here. Or make sure you match `squeezebox2`, too.

Reply:
Then I can go negative and just exclude "boom" from the list - thx